### ISSUE TITLE
Hello,I have a question to this code?

### ISSUE THREAD

**Issue by BCWang93** [1991viet/Yolo-pytorch#11]
Hi,I am very insterested in your code about yolov3.But I find in the 'voc_dataset.py',you use the label is voc mode,like(xmin,ymin,xmax,ymax).But in 'loss.py',when you calculate the gt_box,I find you use the yolo mode,like(xmin,ymin,width,height).But I am not find the transform code in your code.So I want to know how you change the label mode.Thanks!

**Comment by 1991viet** [comment]
Hi,
Thanks for your interest.
Firstly, my implementation is Yolov2, not v3. I changed the label from (xmin,ymin,xmax,ymax) to (xmin,ymin,width,height) at line 125 in the file  **src/data_augmentation.py**

**Comment by BCWang93** [comment]
> Hi,
> Thanks for your interest.
> Firstly, my implementation is Yolov2, not v3. I changed the label from (xmin,ymin,xmax,ymax) to (xmin,ymin,width,height) at line 125 in the file **src/data_augmentation.py**
> Bests
> P/s: I will release my code for Yolov3 soon, but in tensorflow :)

@1991viet  Thanks for your reply!I  get it!Thank you very much!Why do you continued use pytorch to implement the yolov3.I think 'pytorch' is easy and convenient to use compared to 'tensorflow'.

**Comment by 1991viet** [comment]
Hi,
You are right, I admit that pytorch is easier than tensorflow. However, I would like to have more projects in tensorflow for my github account :)
